Makefile

```diff
@@ -13,25 +8,26 @@ MANPREFIX = $(PREFIX)/share/man
 # autoreload backend: inotify/nop
 AUTORELOAD = inotify
 
+# include auto-detected libraries
+include lib_autodetect.mk
+
+# default fallback if lib_autodetect.mk was not found
+HAVE_LIBGIF ?= 1
+HAVE_LIBEXIF ?= 1
+
```

Comment:
If lib_autodetect.mk doesn't exist it will be created at this point. The default vars only matter if they are missing from lib_autodetect.mk and think missing values should air on the safe side (the side more likily to compile) and disable optional libs

Reply:
Do you have any suggestion on how to better word it then?

Reply:
Well the main suggestion was to switch the default, but for the comment
```
# default fallback if vars not found in lib_autodetect.mk
```
Maybe `HAVE_LIBX` instead of vars?

Reply:
I think the updated comment should be clear enough.